components/romio/src/ome/io/nio/PixelsService.java

```diff
@@ -628,9 +633,14 @@ protected String getOriginalFilePath(Pixels pixels)
      */
     protected int getSeries(Pixels pixels)
     {
+        Map<String, String> params = resolver.getPixelsParams(pixels);
+        return getSeries(params);
+    }
+
+    protected int getSeries(Map<String, String> params)
+    {
         try
         {
-            Map<String, String> params = resolver.getPixelsParams(pixels);
             return Integer.valueOf(params.get("image_no"));
```

Comment:
https://ci.openmicroscopy.org/job/OMERO-5.1-merge-deploy-win/ is now green at least.